code/modules/food_and_drink/ingredients.dm

```diff
@@ -780,6 +787,37 @@
 	desc = "An uncooked sheet of pasta."
 	icon_state = "pasta-sheet"
 
+/obj/item/reagent_containers/food/snacks/ingredient/wheat_noodles
+	// generic parent
```

Comment:
kinda pointless comment, if it's meant to be abstract then use `ABSTRACT_TYPE`

Reply:
done'd